app/templates/contacts/new.hbs

```diff
@@ -0,0 +1,119 @@
+        <h2 class="auk-toolbar__title">
+          Niuewe verzendlijst
+        </h2>
+      </Auk::Toolbar::Item>
+    </Auk::Toolbar::Group>
+  </Auk::Toolbar>
+</Auk::Navbar>
+
+<div class="auk-u-m-8">
+  <Auk::ProgressBar @progress={{this.progress}}/>
+  <div class="auk-o-flex auk-o-flex--align-baseline auk-o-flex--justify-between">
+        <h1 class="auk-h1">{{this.title}}</h1>
+        {{#if (and (eq this.step 2) this.isManualInput)}}
+              <Auk::Button
+                @skin="secondary"
+                @icon="add"
+                {{on 'click' this.openContactItemModal}}>
+                contactpersoon toevoegen
+              </Auk::Button>
+        {{/if}}
+  </div>
+  <div>
+    {{#if (eq this.step 0)}}
+      <div class="auk-o-grid-col-6 auk-form-group">
+        <Auk::Label for="name">
+          Titel verzendlijst
+        </Auk::Label>
+        <Auk::Input
+          type="text"
+          id="name"
+          @block={{true}}
+          value={{this.model.name}}
+          {{on "input" (fn this.setInputValue this.model "name")}}
+        />
+      </div>
+    {{/if}}
+    {{#if (eq this.step 1)}}
+      <div class="auk-o-grid-col-6 auk-form-group">
+        <Auk::Label for="name">
+          Kies een input type
+        </Auk::Label>
+        <Auk::Radio
+          @name="inputType"
+          @label="CSV of tabel tekst bestand"
+          @sublabel="Importeer ontvangers van een .csv of .txt bestand"
+          @checked={{(not this.isManualInput)}}
+          {{on 'change' this.setInputType}}
+          />
+        <Auk::Radio
+          @name="inputType"
+          @label="Manuele input"
+          @sublabel="Geef de contact informatie per ontvanger manueel in"
+          @checked={{this.isManualInput}}
+          {{on 'change' this.setInputType}}
+          />
+      </div>
+    {{/if}}
+    {{#if (eq this.step 2)}}
+      <Contact::ContactItemTable
+        @onOpenContactItem={{this.openContactItem}}
+        @model={{this.contacts}}
+        @noDataMessage="Er is nog geen contactpersoon toegevoegd."
+      />
+    {{/if}}
+    <div class="auk-o-grid-col-6 auk-form-group">
+      {{#if (gt this.step 0)}}
+        <Auk::Button
+          @skin="muted"
+          @icon="chevron-left"
+          {{on 'click' this.previousStep}}>
+          Vorige stap
+        </Auk::Button>
+      {{/if}}
+      {{#if (lt this.step 2)}}
+        <Auk::Button
+          @skin="muted"
+          @icon="chevron-right"
+          @layout="icon-right"
+          {{on 'click' this.nextStep}}>
+          Volgende stap
+        </Auk::Button>
+      {{/if}}
+      {{#if (eq this.step 2)}}
+        <Auk::Button
+          @skin="primary"
+          @disabled={{(not this.contacts.length)}}
+          {{on 'click' this.saveContactList}}>
+          Verzendlijst aanmaken
+        </Auk::Button>
+      {{/if}}
+    </div>
+  </div>
+</div>
+{{#if showContactItemModal}}
+  <Contact::ContactItemModal
+    @selectedContact={{this.selectedContact}}
```

Comment:
`@selectedContact` is een beetje vreemd voor een modal die maar 1 contact toont. Gewoon `@contact` volstaat?